example_envs/bookinfo/leafnode.rs

```diff
@@ -32,25 +32,41 @@ impl SimElement for LeafNode {
                 let deq = self.core_node.dequeue(tick);
                 rpc = deq.unwrap();
             } else {
-                // no rpc in the queue, we only react so nothing to do
+                // no rpc in the queue, we only forward so nothing to do
                 continue;
             }
-            // "respond to every node that has sent a request"
+            // forward requests/responses from productpage or reviews
```

Comment:
This is a leafnode, this comment is incorrect.

Reply:
Fixed